src/viser/_gui_api.py

```diff
@@ -190,6 +193,10 @@ def __init__(self) -> None:
         }
         self._current_file_upload_states: Dict[str, FileUploadState] = {}
 
+        self._setup_plotly_js: bool = (
+            False  # Set to True when plotly.min.js has been sent to client.
```

Comment:
nit we can maybe move comment above? this parentheses format is a little bit weird 😛

Reply:
Ok! Moved comment. https://github.com/nerfstudio-project/viser/blob/3af36b1bb606bbb26bd59737c4d44eb6333cecc7/src/viser/_gui_api.py#L196-L199